node-registrar/pkg/db/farms.go

```diff
@@ -41,9 +43,10 @@ func (db *Database) GetFarm(farmID uint64) (farm Farm, err error) {
 
 func (db *Database) CreateFarm(farm Farm) (uint64, error) {
 	if err := db.gormDB.Create(&farm).Error; err != nil {
-		if errors.Is(err, gorm.ErrDuplicatedKey) {
+		if strings.Contains(err.Error(), "duplicate key value") {
```

Comment:
I think that's already handled, are you using latest on development?

Reply:
now yes